### PR TITLE
Delete the old configuration app unused code.

### PR DESCRIPTION
**Pull request by sergibondarenko**
This closes https://floragunn.atlassian.net/browse/ITT-2273

To check this PR
1. Make sure the app is working as expected

### CODE REVIEW

File: public/chrome/configuration/enable_configuration.js [hunk 1]
```diff
@@ -135,24 +133,13 @@ export function enableConfiguration($http, $window, systemstate) {
     systemstate.loadSystemInfo().then(function(){
         // if no REST module is installed the restinfo endpoint is not available, so fail fast
         if (!systemstate.restApiEnabled()) {
-            chrome.getNavLinkById("searchguard-configuration").hidden = true;
+            chrome.getNavLinkById("searchguard-configuration-react").hidden = true;
```
Comment (mgustafsson1) [inline public/chrome/configuration/enable_configuration.js]: We should probably leave out "-react", now that the old app is deleted right?

File: public/chrome/configuration/enable_configuration.js [hunk 2]
```diff
@@ -135,24 +133,13 @@ export function enableConfiguration($http, $window, systemstate) {
...
             return;
         }
         // rest module installed, check if user has access to the API
         systemstate.loadRestInfo().then(function(){
             if (systemstate.hasApiAccess()) {
-                chrome.getNavLinkById("searchguard-configuration").hidden = false;
-                FeatureCatalogueRegistryProvider.register(() => {
```
Comment (mgustafsson1) [inline public/chrome/configuration/enable_configuration.js]: Is this added somewhere else? We probably still need this for the Kibana Home page?

Comment (mgustafsson1) [reply public/chrome/configuration/enable_configuration.js]: I see it now!

File: public/chrome/configuration/enable_configuration.js (hunk 1 above)
Comment (sergibondarenko) [reply public/chrome/configuration/enable_configuration.js]: Find it below

Comment (sergibondarenko) [reply public/chrome/configuration/enable_configuration.js]: Fixed

**Pull request merged**